docs/user/polarization.rst

```diff
@@ -0,0 +1,395 @@
+The quantities of interest in a half polarization measurement are the spin-up and spin-down state
+SANS data sets, denoted :math:`S^{\uparrow}(\lambda)` and :math:`S^{\downarrow}(\lambda)`,
+respectively, which are related to the measured states.
+
+Using full polarization introduces two additional conditions for when the analyzer is set to either
+:math:`0^{\circ}` or :math:`180^{\circ}`. As a result, four SANS data sets are measured, which are
+denoted :math:`S^{00}(\lambda)`, :math:`S^{0\pi}(\lambda)`, :math:`S^{10}(\lambda)` and
+:math:`S^{1\pi}(\lambda)`. By analogy with half polarization, there are four spin state SANS data
+sets of interest in a full polarization experiment, namely :math:`S^{\uparrow\uparrow}(\lambda)`,
+:math:`S^{\uparrow\downarrow}(\lambda)`, :math:`S^{\downarrow\uparrow}(\lambda)` and
+:math:`S^{\downarrow\downarrow}(\lambda)`, that are related to the four measured polarization
+states.
+
+Half polarization - polarized beam without polarization analysis
+----------------------------------------------------------------
+
+Half polarization experiments use a spin flipper with a wavelength-dependent efficiency
+:math:`e(\lambda)` between the polarizer and the sample to rotate the spins from up to down, or vice
+versa depending on the polarizer in use. Consider the case that the flipper is off. The fractions of
+the beam that are spin-up and spin down, denoted :math:`f_{up}` and :math:`f_{down}`, respectively,
+are given by Equations :eq:`spin-up-fraction` and :eq:`spin-down-fraction`, respectively, and
+sample the spin-up scattering :math:`S^{\uparrow}(\lambda)` and spin-down scattering
+:math:`S^{\downarrow}(\lambda)`.
+
+.. math::
+    f_{up} = \frac{N^{\uparrow}}{N^{\uparrow}+N^{\downarrow}} = \frac{R(\lambda)}{1+R(\lambda)}
+    :label: spin-up-fraction
+
+.. math::
+    f_{down} = \frac{N^{\downarrow}}{N^{\uparrow}+N^{\downarrow}} = \frac{1}{1+R(\lambda)}
+    :label: spin-down-fraction
+
+Then, the total scattering measured by the detector for the off state is given by Equation
+:eq:`half-s-off`, and its uncertainty is given by Equation
+:eq:`half-s-off-uncertainty`.
+
+.. math::
+    S^0(\lambda) = \frac{R(\lambda)}{1+R(\lambda)} S^{\uparrow}(\lambda) +
+    \frac{1}{1+R(\lambda)} S^{\downarrow}(\lambda)
+    :label: half-s-off
+
+.. math::
+    (\delta S^0(\lambda))^2 = \left(\frac{R(\lambda)}{1+R(\lambda)}\right)^2 \left(\delta
+    S^{\uparrow}(\lambda)\right)^2 + \left(\frac{1}{1+R(\lambda)}\right)^2 \left(\delta
+    S^{\downarrow}(\lambda)\right)^2
+    :label: half-s-off-uncertainty
+
+When the flipper is on, it rotates the spins in the beam produced by the polarizer by :math:`\pi`
+with an efficiency :math:`e(\lambda)`. The flipped spins are sensitive to the spin-down scattering
+state. The total scattering measured when the flipper is on, and its uncertainty are given by
+Equations :eq:`half-s-on` and :eq:`half-s-on-uncertainty`, respectively.
+
+.. math::
```

Comment:
you must've have a lot of fun transcribing these equations :yum:

Reply:
in hindsight, I could probably have asked around for the original latex :upside_down_face: